src/components/ListHeader/ListHeader.js

```diff
@@ -8,10 +8,12 @@ import styles from './styles.module.css';
 const ListHeader = () => (
   <div className={styles.listHeader}>
     <div className={styles.headerRow}>
-      <Logo className={styles.logo} />
+      <Logo className={styles.logo} title="Logo" />
       <FontAwesomeIcon
         icon={faShareAlt}
         className={styles.shareIcon}
+        title="Share this list"
+        tabIndex={0}
```

Comment:
The title will be read out by screen readers, the tab index tells the screen reader that the item should be selected if the user is using the tab character to get through a page.